query/sharepointsetting/microsoft365_block_onedrive_business_sync.sql

```diff
@@ -0,0 +1,15 @@
+select
+  -- Required Columns
+  tenant_id as resource,
+  case
+    when jsonb_array_length(allowed_domain_guids_for_sync_app) >= 1 then 'ok'
+    else 'alarm'
+  end as status,
+  case
+    when jsonb_array_length(allowed_domain_guids_for_sync_app) >= 1 then 'Allow onedrive for business sync from unmanaged devices.'
```

Comment:
```suggestion
    when jsonb_array_length(allowed_domain_guids_for_sync_app) >= 1 then 'Allow OneDrive for business sync from unmanaged devices.'
```

Please be sure to use correct spellings for services, providers, etc.